todoreact/src/components/TodoPage.js

```diff
@@ -1,70 +1,63 @@
 import React, { useEffect, useState } from 'react';
 
-import api from '../apis/api';
+import { postTodo, putToggleDone, getTodos } from '../apis/todo';
+import { getCategories } from '../apis/categories';
 import Todo from './Todo';
 import AddTodo from './AddTodo';
 
 const TodoPage = () => {
     const [todos, setTodos] = useState([]);
     const [categories, setCategories] = useState([]);
     const [showConfirmTodoText, setShowConfirmTodoText] = useState(false);
+    // move this to AddTodo
```

Comment:
@patrickcuagan this was the only change you suggested that I haven't done. I looked at it, but at the moment AddToDo is more of a dumb component and it made sense to keep more of the logic and api calls here.